### DISCUSSION TITLE
send a specific data by mqtt !!!SOLVED!!

### DISCUSSION THREAD

**Discussion by lisergio** [arendst/Tasmota#14471] [Q&A] [answered]
Hi all.
I am using an esp8266, with tasmota 6.7.1 and a Pzem as a consumption meter. It is working correctly.
And by MQTT it sends the following every 5 seconds approximately.

18:46:54 MQT: tele/medidorconsumo/SENSOR = {"Time":"2022-01-16T18:46:54","ENERGY":{"TotalStartTime":"2021-05-10T18:35:26","Total":1513.911,"Yesterday":0.000,"Today":0.000,"Period":0,"Power":334,"ApparentPower":530,"ReactivePower":412,"Factor":0.63,"Frequency":50,"Voltage":233,"Current":2.280}}

and my question is, if you can change the sent message to send a single data.
I only want to send the value of "Power"

Thanks

**Comment by ascillato** [comment, marked as answer]
Yes, see in the docs Rules, teleperiod triggers and publish command. Thanks.

**Reply by lisergio** [reply]
Thanks a lot!!

I managed to do what I wanted!
As a homebridge, it is not compatible with consumption meter accessories (it only works with the EVE app).

I have come up with a way to be able to read the power of the consumption meter.

First of all we send by MQTT only the value that we want to know.( Make a rule... send if the power is greater than 170w)

Rule1 ON Energy#Power>170 DO publish tele/medidorconsumo/SENSOR2 %value%  ENDON
Rule1 1

Then using MQTTTHING we create a "light" fixture (since it can display values from 0 to more than 10000)

        {
            "accessory": "mqttthing",
            "plugin_map": {
                "plugin_name": "homebridge-mqttthing",
                "index": 0
            },
            "topics": {
                "getCurrentAmbientLightLevel": "tele/medidorconsumo/SENSOR2"
            },
            "type": "lightSensor",
            "name": "Medidor de consumo",
            "url": "http://192.168.0.56"
        }

Now we have a fake accessory that shows us the power in real time (but puts LUX instead of W)

![IMG_3690](https://user-images.githubusercontent.com/289037/149988092-9ec4ded3-b31b-40bb-a47c-f879aec6a4ea.jpg)
